### ISSUE TITLE
[New Rule] write print() instead of print("")

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#134]
## Explanation

It does the same thing and `print()` is shorter.

## Example

```python
# Bad
print("")

# Good
print()
```
